instruqt-tracks/terraform-cloud-azure-v2/13-private-module-registry/assignment.md

```diff
@@ -104,13 +104,11 @@ timelimit: 1800
     font-size: 12px;
     font-weight: bold;
     align-items: center;
-    justify-content: center;
     height: 24px;
   }
 
-  w > img {
+  w > a img {
```

Comment:
Do the other tracks' CSS blocks need to change from `w > img` to `w > a img` as well?

Reply:
All of them are updated.

Reply:
I added the missing tag, @omkensey. Thanks for flagging it.